developers_chamber/qa/checks.py

```diff
@@ -94,3 +99,21 @@ def _run_check(self):
         wrong_import_order_files = set(changed_files) & set([diff.b_path for diff in self._get_unstaged()])
         if wrong_import_order_files:
             raise QAError('Found unsorted import(s) in following file(s):', '\n'.join(wrong_import_order_files))
+
+
+class TestMethodNamesQACheck(QACheck):
```

Comment:
This and code related to this was cherry picked, as it was accidentally removed in this commit: https://github.com/druids/developers-chamber/pull/69/commits/fb1be5672dd31acc8036f415584214aa81074ffc

Reply:
I've only added the `QA_DISALLOWED_TEST_METHOD_NAME_REGEXP` setting.